normandy/selfrepair/static/js/self_repair.js

```diff
@@ -0,0 +1,22 @@
+import Normandy from './normandy_driver.js';
+import {fetchRecipes, filterContext, doesRecipeMatch, runRecipe} from './self_repair_runner.js';
+
+// Actually fetch and run the recipes.
+fetchRecipes().then(recipes => {
+  let context = filterContext().then(context => {
+    // Update Normandy driver with user's country.
+    Normandy._location.countryCode = context.normandy.country;
+
+    for (let recipe of recipes) {
+      doesRecipeMatch(recipe, context).then(([recipe, match]) => {
```

Comment:
And the context object here wasn't guaranteed to be ready yet, so none of the recipes would come back as matches.